README.md

```diff
@@ -233,6 +236,84 @@ should receive PRs for the YAML. The files must end with the `.yml` suffix.
 Workload YAML itself is not currently linted but please try to make the files
 look tidy.
 
+## Patch-Testing Genny Changes with Sys-Perf / DSI
+
+This requires the [evergreen command-line client](https://evergreen.mongodb.com/settings).
+
+Create a `genny` patch-build to compile genny and upload the result to S3.
+(You can skip this if you've got a github PR open: it automatically schedules
+patch-builds.)
+
+```sh
+cd genny
+evergreen patch -p genny
+```
+
+That will output something like this:
+
+```
+	     ID : 5c533a2732f4174bbcb8bb2e
+	Created : 35.656ms ago
+    Description : <none>
+	  Build : https://evergreen.mongodb.com/patch/5c533a2732f4174bbcb8bb2e
+      Finalized : No
+```
+
+Navigate to the build URL and schedule `t_compile` on the Amazon Linux 2
+variant. Clever folks could maybe put this all into the `evergreen patch`
+invocation.
+
+Wait for the task to complete and copy the URL for the "Genny Patch" File. It
+will look similar to this:
+
+```
+https://s3.amazonaws.com/mciuploads/genny/patches/genny_amazon2_patch_ca6805464c1ef7e580496070beea97c88277f067_5c533a2732f4174bbcb8bb2e_19_01_31_18_11_51/genny.tgz
+```
+
+Now create a sys-perf patch:
+
+```sh
+cd mongo
+evergreen patch -p sys-perf
+```
+
+This will give you a different Build URL.
+
+Now modify DSI to point to your Genny binary. Modify
+`workload_setup.common.yml` to look like the following:
+
+```yaml
+  # Get Genny's distribution
+  - on_workload_client:
+      exec: |
+        set -eou pipefail
+
+        cd genny
+
+        genny_revision="$(git rev-parse HEAD)"
+        ....
+        # genny_custom_url="${bootstrap.genny_custom_url}"
+        genny_custom_url="--- >>> PUT YOUR URL HERE <<<---"
```

Comment:
Could you add a note to [TIG-1408](https://jira.mongodb.org/browse/TIG-1408) to update these instructions?